src/components/Form/Form.tsx

```diff
@@ -0,0 +1,66 @@
+import { Form as AntForm, FormProps as AntFormProps, FormInstance } from 'antd'
+import { PropsWithChildren, ReactElement, ReactNode, useMemo } from 'react'
+
+import { Button } from '../Button'
+
+type FormProps<Values> =
+  PropsWithChildren<AntFormProps<Values>>
+  & {
+  columns?: number
+  gap?: number
+  submitButton?: boolean | string | ((form: FormInstance) => ReactNode)
+}
+
+const defaultSubmitButton = (label: ReactNode = 'Submit'): ReactNode => {
+  return (
+    <Button htmlType={Button.HTMLType.Submit}>
+      {label}
+    </Button>
+  )
+}
+
+const renderSubmitButton = (
+  form: FormInstance,
+  submitButtonProp: boolean | string | ((form: FormInstance) => ReactNode)
+): ReactNode => {
+  switch (typeof submitButtonProp) {
+  case 'boolean':
+    return submitButtonProp && defaultSubmitButton()
+  case 'string':
+    return defaultSubmitButton(submitButtonProp)
+  case 'function':
+    return submitButtonProp(form)
+  default:
```

Comment:
We don 't accept other types, I would throw error

Reply:
This line is outdated, the new implementation is here, and the default is handled correctly:

https://github.com/mia-platform/design-system/pull/734/files#diff-7fa3d059e7802fedcc7a6d434f187ada28cb4aa1a92a95e0ccbf41935179ec26